secp256k1/abi.nim

```diff
@@ -223,7 +223,7 @@ var secp256k1_nonce_function_default*: secp256k1_nonce_function
 ## Recovery interface follows
 
 type
-  secp256k1_ecdsa_recoverable_signature* = object
+  secp256k1_ecdsa_recoverable_signature* {.bycopy.} = object
```

Comment:
why the `bycopy` here?

Reply:
Oh, you're right. Looks like it's only ever passed with a pointer. Shouldn't need `bycopy` then.